block/tipset_test.go

```diff
@@ -1,42 +1,44 @@
-package types
+package block
```

Comment:
This should be block_test. I think that will then require moving `RequireNewTipSet` back to a block/testing.go file.

Reply:
I moved this into `block_test` since this is the only test that requires it, I'm leaving `RequireNewTipSet` in `tipset_test.go`.